### PR TITLE
Update platform version to `0.36.0-alpha.1`

### PR DESCRIPTION
**Pull request by Neeharika-Sompalli**
- Update platform version to `0.36.0-alpha.0`
- Some changes related to platform APIs
- We no more need to override `int()` and `getPlatformAddressBook` as per new platform contract
- Now platform keeps address Book . So no need to have `ADDRESS_BOOK` child in ServicesState
But, to not shuffle children changed name to `LEGACY_ADDRESS_BOOK`
- Some serialization format for `FCQueue` changed, but doesn't directly affect Services as discussed with @cody-littley. So had to change `MerkleNetworkContextSerdeTest` related hex files

### CODE REVIEW

**Comment by mhess-swl** [comment]
Closes #4817

**Review by mhess-swl — APPROVED** [review]

**Comment by jeromy-cannon** [comment]
release engineering approves the build changes, please wait for other approvers for remaining code before merging.

**Review by jeromy-cannon — APPROVED** [review]

**Review by iwsimon — APPROVED** [review]
LGTM

File: hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/state/migration/StateChildIndices.java
Comment (Neeharika-Sompalli) [inline hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/state/migration/StateChildIndices.java:30]: We can add a comment to say `this address book is provided by platform now. And we are just using this as placeholder to not shuffle children.`

**Pull request merged**
